notebooks/community/vertex_endpoints/optimized_tensorflow_runtime/t5x_base_optimized_online_prediction.ipynb

```diff
@@ -0,0 +1,1551 @@
+{
```

Comment:
<p>remove future tense: "will"</p><p>Replace "we" with "you"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1453/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>update whole doc, removed future tense and change language to avoid using "we".</p>